cve_bin_tool/cvedb.py

```diff
@@ -293,7 +293,8 @@ def populate_db(self) -> None:
             if source_name != "NVD" and cve_data[0] is not None:
                 if source_name != "RSD":
                     cve_data = self.update_vendors(cve_data)
-                cve_data = self.filter_duplicate(cve_data, source_name)
+                # Temporary disable removing duplicates.
+                # cve_data = self.filter_duplicate(cve_data, source_name)
```

Comment:
Did you mean to leave these in?  Looks like they aren't needed any more.

Reply:
@terriko I disabled the duplicate check to see what happenned. It results in more CVEs being reported so it indicated that the duplicate processing seemed to remove more CVEs than was necessary. #2332 posed the question that duplicate entry processing should perhaps be removed from the data loading phase to become part of the output processing but so far I havn't seen any duplicate entries being reported. I will remove the code for now.

Will have a look at mocking some of the testing but I can certainly add some tests to work with some of the downloaded data files.

I don't like the way disabling is done either and it took me a while to work out how to get it to work - so it clearly isn't obvious (see #2332 again) I will change the processing to be the same way that checkers are removed so at least it is consistent.

Having added some extra checks to make the processing more robust, I think RSD needs to removed as none of the data seems to be loading now. As the data isn't being updated either, I don't think it is a loss given the fact we have a direct link now with the RedHat data source.